tests/test_whitelist_validation.py

```
@@ -13,154 +16,155 @@ def test_settings():
     """Test settings with temporary whitelist path."""
     return {
         "whitelist": {"storage_path": "/tmp/test_validation_whitelist.json"},
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_validation_whitelist.json"

(S108)